ExtLibs/Maps/GMapMarkerPlane.cs

```diff
@@ -165,20 +165,30 @@ public override void OnRender(IGraphics g)
             // the plane
             g.TranslateTransform(-2, -2);
 
+            var fill_color = Color.White;

```

Comment:
white ?

Reply:
You're right, this should be cleaned up. Basically, this initializer is unnecessary, as it is guaranteed to get overwritten in the next few lines. it's just there to shut up the code analysis from complaining that fill_color may be uninitialized (even though it's guaranteed to be).

I'll clean this up in my upstream PR. I'll leave this as is here for now.